ground/gcs/src/inc/widgetnames.h

```diff
@@ -1,6 +1,18 @@
 #ifndef WIDGETNAMES_H
 #define WIDGETNAMES_H
 
-
+#define CONFIGGADGETWIDGET_SIDEBAR						"configGadgetSideBar"
+#define CONFIGGADGETWIDGET_SIDEBAR_VBOXLAYOUT			"configGadgetSideBarVBox"
+#define CONFIGGADGETWIDGET_HWSETTINGS					"configGadgetHwSettings"
+#define CONFIGGADGETWIDGET_VEHICLETYPE					"configGadgetVehicleType"
+#define CONFIGGADGETWIDGET_INPUT						"configGadgetInput"
+#define CONFIGGADGETWIDGET_OUTPUT						"configGadgetOutput"
+#define CONFIGGADGETWIDGET_ATTITUDE						"configGadgetAttitude"
+#define CONFIGGADGETWIDGET_STABILIZATION				"configGadgetStabilization"
+#define CONFIGGADGETWIDGET_MODULES						"configGadgetModules"
+#define CONFIGGADGETWIDGET_AUTOTUNE						"configGadgetAutotune"
+#define CONFIGGADGETWIDGET_CAMERASTAB					"configGadgetCameraStab"
+#define CONFIGGADGETWIDGET_TXPID						"configGadgetTxPID"
+#define CONFIGGADGETWIDGET_OSD							"configGadgetCameraOSD"
 
```

Comment:
Right now it only contains the config widgets, because I haven't looked much into the other code files yet.

The principal idea was to make objects addressable by name from a global stylesheet. Say if I were to want to put a background image into the config widget iconbar, without name, the selector I could use would be either MyTabbedStackWidget > QListWidget and I'd have to pray that no one else uses that same nested order, or use the the whole hierarchy and pray that no one changes it during UI work. With naming a stylesheet gets maintainable and selectors stay short. And as you suggested, it'd likely help with crashdumps.

The idea with widgetnames.h was mainly to have all names in a central place, so that you don't need to run after each code file, if you have/need to rename the object names for whatever reason.